composer.json

```diff
@@ -29,8 +29,7 @@
     },
     "suggest": {
         "quartzy/courier-sparkpost": "To deliver emails using Sparkpost",
-        "sendgrid/sendgrid": "To use the SendGridCourier",
-        "wildbit/postmark-php": "To use the PostmarkCourier"
```

Comment:
Should this suggest camuthig/courier-postmark as well?

Reply:
It should. Thanks for the catch.